controllers/openshift_controller.go

```diff
@@ -751,12 +751,8 @@ func (r *KataConfigOpenShiftReconciler) createRuntimeClass(runtimeClassName stri
 		}
 	}
 
-	if r.kataConfig.Status.RuntimeClass == "" {
-		r.kataConfig.Status.RuntimeClass = runtimeClassName
-		err = r.Client.Status().Update(context.TODO(), r.kataConfig)
```

Comment:
I'm fine with this change as it partly addresses https://issues.redhat.com/browse/KATA-2441.